donkeycar/parts/oled.py

```diff
@@ -81,7 +81,7 @@ def display_text(self):
                 self.canvas.text((x, top), text, font=self.font, fill=255)
                 top += 8
 
-    def run(self, recording, num_records, user_mode):
+    def run_threaded(self, recording, num_records, user_mode):
```

Comment:
Look at the old `OLEDPart`. The threaded part needs to run in its own thread, and therefore should define a `run` method.  Then `run_threaded` can update the internal state of the car, and dispatch a call to `update`. I think you should also make `roundUp` optional.

Reply:
Ah, I see the problem here with my understanding. I will fix this in the next iteration until the end of the week